content/graphql/advanced.md

```diff
@@ -1,3 +1,22 @@
+## Cache Implementation
```

Comment:
there's a double "Cache" title in the deployed env, it looks weird, not sure exactly why but we should keep just one.

Reply:
I had a double title, I removed it